components/activity_and_insights/insights/card/card.tsx

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React, {memo} from 'react';
+import classNames from 'classnames';
+
+import Icon from '@mattermost/compass-components/foundations/icon/Icon';
+
+import Card from 'components/card/card';
+import CardHeader from 'components/card/card_header';
+import {CardSize, CardSizes} from '../insights';
+
+import './card.scss';
+
+type Props = {
+    class: string;
```

Comment:
Yeah, I mean just the purpose of the `class` prop for a component user (developer). It's to customize the styling. Does specifying the class name need to be part of the component API or could we generate a unique style name that can be optionally styled against? Is more or less params better in this case?

Reply:
I can look into generating the classname by trying to utillize: https://reactjs.org/docs/react-component.html#displayname. That could reduce the number of props needed.

Having these props won't dramatically impact performance but the less we have the easier it could be to use. I will also have to consider readability/complexity by doing this but we can assess once it's implemented

Reply:
I don't have a strong opinion on this, just things for you to consider in your choice.

Reply:
I'm going to leave it as is for now but I might revisit it